test/transforms.jl

```diff
@@ -1277,60 +1277,88 @@
   end
   
   @testset "Levels" begin
-    a = categorical(["yes", "no", "no", "no", "yes"]) 
-    b = categorical([1, 2, 4, 2, 8], ordered=false) 
-    c = categorical([1, 2, 1, 2, 1]) 
-    d = [1, 23, 5, 7, 7]
-    e = categorical([2, 3, 1, 4, 1])
-    t = Table(; a, b, c, d, e)
```

Comment:
Please do not change existing tests unless strictly necessary. Your changes should be backward compatible right? You are changing the input tables for the tests, which requires a closer look and review. If you can undo these changes I can more easily review the additions being certain that nothing that used to work is breaking.

Reply:
I added and updated the old tests. I removed tests that checked for incorrect behavior of the `ordered` keyword and added more tests.